types/node/tls.d.ts

```diff
@@ -285,7 +405,29 @@ declare module "tls" {
     }
 
     class Server extends net.Server {
+        /**
+         * The server.addContext() method adds a secure context that will be
+         * used if the client request's SNI name matches the supplied hostname
+         * (or wildcard).
+         */
         addContext(hostName: string, credentials: SecureContextOptions): void;
+        /**
+         * Returns the session ticket keys.
+         */
+        getTicketKeys(): Buffer;
+        /**
+         *
```

Comment:
Nit: Remove this line

Reply:
What do you mean? https://nodejs.org/dist/latest-v12.x/docs/api/tls.html#tls_server_getticketkeys says this is real method and my test proves that it returns a `Buffer` indeed.